packages/searchbase/src/index.js

```diff
@@ -892,8 +911,8 @@ class Searchbase {
     );
   }
 
-  getAppbaseSettings(): {| recordAnalytics: boolean |} {
-    return { recordAnalytics: this.analytics };
+  getAppbaseSettings(): AppbaseConfig {
```

Comment:
@ShahAnuj2610 RS API settings are not similar to `appbaseConfig`, what if `appbaseConfig` has some new properties?